mattermost-plugin/webapp/src/index.tsx

```diff
@@ -233,13 +233,6 @@ export default class Plugin {
             }
 
             this.registry.registerPostWillRenderEmbedComponent((embed) => embed.type === 'boards', BoardsUnfurl, false)
-        } else {
-            windowAny.frontendBaseURL = subpath + '/plug/focalboard'
-            this.channelHeaderButtonId = registry.registerChannelHeaderButtonAction(<FocalboardIcon/>, () => {
-                const currentChannel = mmStore.getState().entities.channels.currentChannelId
-                window.open(`${window.location.origin}/plug/focalboard/workspace/${currentChannel}`)
-            }, 'Boards', 'Boards')
-            this.registry.registerCustomRoute('/', MainApp)
```

Comment:
This section is never called. The initialize function is called by the web app - here 
https://github.com/mattermost/mattermost-webapp/blob/master/plugins/index.js#L172

The PluginRegistry is created with a "registerProduct" function. I'm leaving the check as a sanity check, but the else section is not needed and is wrong as well.